### ISSUE TITLE
Flowplayer playback in demo doesn't work on macOS Safari

### ISSUE THREAD

**Issue by mrlika** [Novage/p2p-media-loader#21]

**Comment by mrlika** [comment]
> In version 0.3.0, It seems not stable in flowplayer with Safari

Fixed. No need to change P2P Engine code because it is just Flowplayer configuration issue. I will update the demo and the samples soon.

```JS
var player = flowplayer("#video", {
                autoplay: true,
                muted: true,
                clip: {
                    sources: [{
                        type: "application/x-mpegurl",
                        src: this.videoUrl
                    }]
                },
                hlsjs: {
                    liveSyncDurationCount: this.liveSyncDurationCount,
                    loader: this.engine.createLoaderClass(),
                    safari: true
                }
            });
```

safari
boolean
If set to true the plugin is enabled in Safari. Please read the section on browser support before enabling this option. Default: false

> peer.remoteAddress is undefined
Yes. It is the default behavior of Safari for security reasons.

**Comment by mrlika** [comment]
Fixed in 53ab49f.

WebRTC implementation in Safari is still buggy. Very often WebRTC stops working in Safari (no peers). Reopening page in new tab doesn't help. You have to restart the browser in this case.
